runtime/compiler/aarch64/codegen/ARM64JNILinkage.cpp

```diff
@@ -1059,19 +1053,6 @@ TR::Register *J9::ARM64::JNILinkage::buildDirectDispatch(TR::Node *callNode)
    TR::RegisterDependencyConditions *deps = new (trHeapMemory()) TR::RegisterDependencyConditions(maxRegisters, maxPostRegisters, trMemory());
 
    size_t spSize = buildJNIArgs(callNode, deps, passThread, passReceiver, killNonVolatileGPRs);
```

Comment:
`spSize` will be unused after the change.

Reply:
Removed `spSize` variable.